### PR TITLE
Add note about installer

### PR DESCRIPTION
**Pull request by chamons**

### CODE REVIEW

File: docs/getting-started-objective-c.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ Your next step is to download and install the embeddinator on your Mac.
 
 There is no package to download yet. Until one is available you need to build it from our git repository, see the [constributing](Contributing#Objective-C) document for instructions.
 
+Installers can be built by running "make installer" in the objcgen directory.
```
Comment (spouliot) [inline docs/getting-started-objective-c.md]: that should be mentioned in the Contributing document - not in the getting started :)

File: docs/getting-started-objective-c.md [hunk 2]
```diff
@@ -28,6 +28,9 @@ Your next step is to download and install the embeddinator on your Mac.
...
+
+Once installed via an installer, you can run the tool via /Library/Frameworks/Xamarin.Embeddinator-4000/Commands/objcgen
```
Comment (spouliot) [inline docs/getting-started-objective-c.md]: Before this just add a quick walkthru of the installer

**Comment by chamons** [comment]
Better @spouliot ? I screen shotted the installer, which might be overkill but is easy.

File: docs/Contributing.md [hunk 3]
```diff
@@ -27,6 +27,7 @@ $ mono objcgen/bin/Debug/objcgen.exe
 
 Tests can be executed by running `make` from the `tests/objc-cli` directory.
 
+Installers can be built by running "make installer" in the objcgen directory.
```
Comment (spouliot) [inline docs/Contributing.md]: Installers (plural), maybe _The installer..._ ?

Comment (spouliot) [inline docs/Contributing.md]: objcgen -> `objcgen` (to match above line)

**Pull request merged**